test/suite/test_rollback_to_stable01.py

```diff
@@ -26,18 +26,41 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import wiredtiger, wttest
+import os, subprocess, wiredtiger, wttest
 from wtdataset import SimpleDataSet
 from wiredtiger import stat, wiredtiger_strerror, WiredTigerError, WT_ROLLBACK
 from wtscenario import make_scenarios
 from time import sleep
 
+def get_git_root():
```

Comment:
This seems a bit odd to me - I'd rather not assume that the test is always run from a Git checkout. Is there another way to achieve this?

Reply:
I had a think, unfortunately I think looking for `../../tools/etc` is the cleanest way to do it. I thought about `find .. -name rts_verify` or similar, and there wasn't anything that felt better. I've made this change in c5a7072, let me know what you think.